Assets/Scripts/Player.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Player : MonoBehaviour {
+
+    public float speedMultiplier;
+    public float sprintMultiplier;
+    public float jumpMultiplier;
+    public float jumpCooldown;
+    public float throwForce = 45f;
+
+    public Crosshairs crosshairs;
+    public GameObject grenadePrefab;
+    public GameObject AreaOfEffectType;
```

Comment:
Both grenade and areaOfEffectType may not be used per character. These would be added in the prefab stage of character building right?